thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -363,6 +369,46 @@ export default Ember.Controller.extend({
     };
   },
 
+  /**
+   * Fetches the current session state from the backend and issues a notification if it has been updated
+   *
+   * @private
+   */
+  _checkSession() {
+    const { sessionId, sessionUpdatedTime } = this.getProperties('sessionId', 'sessionUpdatedTime');
+
+    if (!sessionId) { return; }
+
+    fetch(`/session/${sessionId}`)
+      .then(checkStatus)
+      .then(res => {
```

Comment:
nit: let's use parens `(res)` when the arrow function spans over multiple lines (our convention)